app/views/admin/stats/referrals/index.html.slim

```diff
@@ -0,0 +1,14 @@
+div.panel-body
+  div.adv-table
+    table.display.table.table-bordered.table-striped.dynamic-table id="dynamic-table"
+      tr
+        th Channel
+        th Earnings
+        th Publisher
+      tbody
+        - @result.each do |entry|
+          - publisher_id = entry['account_id'].split(":")[1]
```

Comment:
You can use this method to find the publisher given the id with the prefix in order to display name, a link to their admin page, as well as grab other information you might want to display https://github.com/brave-intl/publishers/blob/1e80e15b001374ab8e7b2697c68bebce182ca095/app/models/publisher.rb#L328-L330

Reply:
I'll add this.

Reply:
We can also use the *_channel_details.url method to display the url for any channel type.  https://github.com/brave-intl/publishers/blob/1e80e15b001374ab8e7b2697c68bebce182ca095/app/models/youtube_channel_details.rb#L32-L34

Reply:
you can probably avoid a n+1 query with eager loading in the controller.